### PR TITLE
upgraded byte-buddy

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -227,7 +227,7 @@
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
     <version.org.wildfly.swarm>2018.3.3</version.org.wildfly.swarm>
-    <version.net.byte-buddy>1.4.16</version.net.byte-buddy>
+    <version.net.byte-buddy>1.8.17</version.net.byte-buddy>
```
Comment (mareknovotny) [inline pom.xml]: do we know why we manage byte-buddy?
just got from another change that it is used in mockito and optaweb-employee-rostering override managed version due this https://github.com/kiegroup/optaweb-employee-rostering/blob/master/pom.xml#L86

Comment (mbiarnes) [reply pom.xml]: I can't find a dependency of org.mocikito:mockito-core: [1.1.0.19](https://search.maven.org/artifact/org.mockito/mockito-core/1.10.19/jar) to net.byte-buddy

**Comment by mbiarnes** [comment]
this PR has to be backported to 7.14.x

**Comment by mareknovotny** [comment]
> this PR has to be backported to 7.14.x

why ?

**Comment by mbiarnes** [comment]
@mareknovotny I thought the 7.2 has to be aligned to JBEAP 7.2 - I removed the comment

File: pom.xml (hunk 1 above)
Comment (mareknovotny) [reply pom.xml]: I am talking about 2.x look at https://repository.jboss.org/nexus/index.html#nexus-search;gav~org.mockito~mockito-core~~~~kw,versionexpand%20
That is used in https://github.com/kiegroup/optaweb-employee-rostering/blob/master/pom.xml

**Comment by mbiarnes** [comment]
will close this PR until it is decided to upgrade or to remove net.byte-buddy

**Pull request closed without merging**